libs/application/ui-fields/src/lib/RedirectToServicePortalFormField/RedirectToServicePortalFormField.tsx

```diff
@@ -12,7 +12,7 @@ interface Props extends FieldBaseProps {
 
 export const RedirectToServicePortalFormField: FC<
   React.PropsWithChildren<Props>
-> = ({ application }) => {
+> = ({ field, application }) => {
```

Comment:
What do you think, would  `{ field: {marginTop = 10, marginBottom = 10}, application }` be too messy?

Reply:
A little, but I've fixed this file to also use deconstruction